### PR TITLE
[SW-1205] Fix formating in pysparkling package

### PR DESCRIPTION
**Pull request by jakubhava**
No semantic changes -> just clicked on reformat code in Idea. Will help us in the future to avoid unnecessary conflicts

### CODE REVIEW

File: py/pysparkling/__init__.py [hunk 1]
```diff
@@ -24,18 +24,19 @@
...
 
+warnings.formatwarning = custom_formatwarning
 
 if not (pyspark_version[0] == pysparkling_version[0] and pyspark_version[1] == pysparkling_version[1]):
     warnings.warn("""
     You are using PySparkling of version {pysparkling_version}, but your PySpark is of
-    version {pyspark_version}. Please make sure Spark and PySparkling versions are compatible. """.format(pysparkling_version=__version__, pyspark_version=pyspark.__version__))
-
+    version {pyspark_version}. Please make sure Spark and PySparkling versions are compatible. """.format(
+        pysparkling_version=__version__, pyspark_version=pyspark.__version__))
```
Comment (mn-mikke) [inline py/pysparkling/__init__.py]: I would prefer the original version.

File: py/pysparkling/context.py [hunk 2]
```diff
@@ -113,14 +131,15 @@ def __default_h2o_connect(h2o_context, **kwargs):
...
+            url = "{}://{}:{}/{}".format(schema, h2o_context._client_ip, h2o_context._client_port,
+                                         h2o_context._conf.context_path())
             return h2o.connect(url=url, **kwargs)
         else:
             return h2o.connect(ip=h2o_context._client_ip, port=h2o_context._client_port, **kwargs)
 
-
     @staticmethod
-    def getOrCreate(spark, conf=None, verbose=True, pre_create_hook=None, h2o_connect_hook=__default_h2o_connect, **kwargs):
+    def getOrCreate(spark, conf=None, verbose=True, pre_create_hook=None, h2o_connect_hook=__default_h2o_connect,
```
Comment (mn-mikke) [inline py/pysparkling/context.py]: I would prefer the original version.

File: py/pysparkling/context.py [hunk 3]
```diff
@@ -271,7 +290,8 @@ def as_h2o_frame(self, dataframe, framename=None, full_cols=100):
                     return fc._as_h2o_frame_from_RDD_String(self, dataframe, framename, full_cols)
                 elif isinstance(first, bool):
                     return fc._as_h2o_frame_from_RDD_Bool(self, dataframe, framename, full_cols)
-                elif (isinstance(dataframe.min(), int) and isinstance(dataframe.max(), int)) or (isinstance(dataframe.min(), long) and isinstance(dataframe.max(), long)):
+                elif (isinstance(dataframe.min(), int) and isinstance(dataframe.max(), int)) or (
+                        isinstance(dataframe.min(), long) and isinstance(dataframe.max(), long)):
```
Comment (mn-mikke) [inline py/pysparkling/context.py]: I would start the line with ```(```

File: py/pysparkling/context.py [hunk 4]
```diff
@@ -113,14 +131,15 @@ def __default_h2o_connect(h2o_context, **kwargs):
         if schema == "https":
             kwargs["https"] = True
         if h2o_context._conf.context_path() is not None:
-            url = "{}://{}:{}/{}".format(schema, h2o_context._client_ip, h2o_context._client_port, h2o_context._conf.context_path())
+            url = "{}://{}:{}/{}".format(schema, h2o_context._client_ip, h2o_context._client_port,
```
Comment (mn-mikke) [inline py/pysparkling/context.py]: I would prefer the original version.

File: py/pysparkling/initializer.py [hunk 5]
```diff
@@ -91,11 +108,12 @@ def check_different_h2o():
...
             command_import_h2o = "import h2o; print(h2o.__version__)"
             full_command = "python -c '" + command_sys_path + command_import_h2o + "'"
-            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode('utf-8').replace("\n", "")
+            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode(
+                'utf-8').replace("\n", "")
             if not previous_version == sw_h2o_version and previous_version is not "":
                 warnings.warn("PySparkling is using internally bundled H2O of version " +
                               str(sw_h2o_version) + ", but H2O installed in the python"
-                              " environment is of version " + str(previous_version) + ".")
+                                                    " environment is of version " + str(previous_version) + ".")
```
Comment (mn-mikke) [inline py/pysparkling/initializer.py]: What about multiline string interpolation?

File: py/pysparkling/initializer.py [hunk 6]
```diff
@@ -91,11 +108,12 @@ def check_different_h2o():
             command_sys_path = "import sys; sys.path = " + str(path_without_sw).replace("'", "\"") + ";"
             command_import_h2o = "import h2o; print(h2o.__version__)"
             full_command = "python -c '" + command_sys_path + command_import_h2o + "'"
-            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode('utf-8').replace("\n", "")
+            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode(
```
Comment (mn-mikke) [inline py/pysparkling/initializer.py]: What about this?:
```
subprocess
   .check_output(full_command, shell=True, stderr=DEVNULL)
   .decode('utf-8')
   .replace("\n", "")
```

**Comment by jakubhava** [comment]
Thx for review @mn-mikke . Closing for now as we don't have time to work on custom formatting template

**Pull request closed without merging**